### PR TITLE
SIGNIFICANTLY buffs c20r and akl reavers (see inside for reasoning)

### PR DESCRIPTION
**Pull request by nikothedude**
## About The Pull Request
Buffs c20r reavers by upping their burst size from 3 to 8, and bufs akl reavers by increasing their burst size from 3 to 5 and making them do 10% extra damage. Every change is reflected by their gun getting modded the fuck out.

The AKL reaver also actually uses an AKL instead of a firestorm.

I like this change because reavers are supposed to be the voidwolf /elite/, theyre supposed to be given significantly better equipment. yet somehow both of these reavers are equipped significantly worse than the voidwolf trooper.

Plus, reavers started out as, as I was told, "scary event mobs" that were never meant to be mapped. So, I'm treating them as that. 

This makes reavers kind of like, really scary. They can fuck you /up/, easily. Thankfully they only exist in one place on the map, but this will probably make most people not go there. Not that that's terrible.

As usual this also has misc code improvements.

image

Map changes: Added 1 c20r reaver to the bathroom. Added one medkit to the first room along with a crate of drugs, a few voidsuits as well. In the second room, added a bunch of tables with loot on them. Tables go as follows: Top two: 4 packets of laser rifle ammo, 3 loaded laser rifle mags. 3 packets of 9mm, 2 9mm HV smg mags. Side tables go as follows: A bunch of random armor spawns with 3 guaranteed xanorath voidsuits and some armor mods. A b unch of random voidwolf mods. 5 energy cutlasses and 2 random melee weapons. A bunch of random high-to-medium tier guns. 2 guaranteed bluecross guns. 3 smoke grenades and a bunch of random high-tier ammo. In the bathroom: A guaranteed reaver SCAF suit. 3 racks, as follows: 1 with a fuckton of cleaner grenades. One with 1 high-yield EMP grenades, 2 low-yield EMp grenades,  3 heatwave grenades, 5 flashbangs. One with 3 frag grenades, 5 HE grenades, 2 stinger grenades, and 1 incendiary grenade. Yes, I know these are contraband.

## Changelog
:cl:
balance: Voidwolf c20r and akl reavers have been significantly buffed.
/:cl:

### CODE REVIEW

**Comment by nikothedude** [comment]
i want this reviewed before its opened

File: code/modules/mob/living/carbon/superior_animal/human/subtype/voidwolf/voidwolf.dm [hunk 1]
```diff
@@ -216,35 +211,68 @@
...
+
+	rapid_fire_shooting_amount = 5 //we're using the burst 5 mode
+	delay_for_rapid_range = 0.22 SECONDS
+
+/obj/item/gun/projectile/automatic/ak47/akl/reaver_modded
+
+	initialized_upgrades = list(/obj/item/gun_upgrade/trigger/dangerzone,
+								/obj/item/tool_upgrade/refinement/laserguide,
+								/obj/item/tool_upgrade/productivity/ergonomic_grip,
+								/obj/item/gun_upgrade/barrel/bore,
```
Comment (ThePainkiller) [inline code/modules/mob/living/carbon/superior_animal/human/subtype/voidwolf/voidwolf.dm:251]: While I very much appreciate the fact the gun has an actual damage mod to justify the 10% increase in damage, this is still a guild-crafted only item. Why would wolves have this?
I would suggest replacing it with a void wolf gauss coil for the sake of a theme, also making them significantly more worth killing

Comment (cdb-is-not-good) [reply code/modules/mob/living/carbon/superior_animal/human/subtype/voidwolf/voidwolf.dm:251]: the heavy bore is made by the guild but it's not special in any way. It's just a handmade barrel.

Comment (cdb-is-not-good) [reply code/modules/mob/living/carbon/superior_animal/human/subtype/voidwolf/voidwolf.dm:251]: It's not one of their 'signature' items like the scope that IS infact really, really well made.

Comment (nikothedude) [reply code/modules/mob/living/carbon/superior_animal/human/subtype/voidwolf/voidwolf.dm:251]: The heavy barrel is literally the only thing that increases damage at all. I could do some funky stuff with the gaus barrel though.

Maybe later though. Im gonna play divinity.

**Comment by nikothedude** [comment]
aight ill map in some extra rewards

**Comment by cdb-is-not-good** [comment]
Looks good. May need further adjustments given how much of a reward this place is now but in its current state i'm rather happy with it.

**Comment by Stutternov** [comment]
just - noticed.. the alk is useless for them to even use/drop. the gun is only printable by soteria as well as its ammo.

**Pull request merged**